include/slang/driver/Driver.h

```diff
@@ -73,6 +73,9 @@ class SLANG_EXPORT Driver {
         /// A set of extensions that will be used to exclude files.
         flat_hash_set<std::string> excludeExts;
 
+        /// A set of preprocessor directives to be ignored
```

Comment:
Small nit, can you end these comments with a period to match the other comments in this file and elsewhere?

Reply:
Done.